.vscode/settings.json

```diff
@@ -1,6 +1,10 @@
 {
-    "cSpell.words": [
-        "protos"
-    ],
-    "editor.formatOnSave": true,
-}
\ No newline at end of file
+  "cSpell.words": ["protos"],
```

Comment:
without the platform-io/arduino content, do we need to be sharing a .vscode folder?

Reply:
I like including the .vscode settings for the sake of helping developers setup their IDE for this project specifically. For example the default prettier settings conflict with the rules defined in the project (single quotes, and semicolons come to mind).

(In ref to the specific line you pointed out) Im also terrible at spelling so having the spelling plugin installed helps a bunch.

Reply:
That's fair, but I wonder if there's some other way to enforce those kind of deps, like ci checks. I'm actually not using VSCode much anymore & I don't think JetBrains things respect the .vscode folder.